Shared/ThresholdKeyView.swift

```diff
@@ -149,40 +142,35 @@ struct ThresholdKeyView: View {
                                     }
                                     shareCount += 1
                                 } while !finishedFetch
-
                                 // There are 0 locally available shares for this tkey
                                 if shareCount == 0 {
-                                    // Attempt reconstruction
-                                    guard let reconstructionDetails = try? await threshold_key.reconstruct() else {
-                                        alertContent = "Failed to reconstruct key. \(threshold) more share(s) required"
+                                    var shareIndexes = try threshold_key.get_shares_indexes()
+                                    shareIndexes.removeAll(where: {$0 == "1"})
```

Comment:
You should attempt reconstruction first
```suggestion
guard let reconstructionDetails = try? await threshold_key.reconstruct() else {
                                        alertContent = "Failed to reconstruct key. \(threshold) more share(s) required"
                                        resetAccount = true
                                        showAlert = true
                                        return
                                        }
                                        
                                    var shareIndexes = try threshold_key.get_shares_indexes()
                                    shareIndexes.removeAll(where: {$0 == "1"})
```